.changeset/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "bigcommerce/catalyst" }
+  ],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
```

Comment:
Docs: [`fixed`](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#fixed-array-of-arrays-of-package-names)
Docs: [`linked`](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#fixed-array-of-arrays-of-package-names)

I don't believe that we have a need for these two properties, so I'm leaving them blank. The only idea I had is if we wanted to tie the version of Storybook to the current version of the component library, but I don't think that every new version of the component library would always result in a new version of Storybook.

Reply:
We're still somewhat undecided on this point; the ultimate decision comes down to whether or not we choose to version + tag Storybook `apps/docs`, for example.